### PR TITLE
Readability improvements - Fundamentals data scheduled - publishing

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Belongs to: https://github.com/umbraco/UmbracoDocs/issues/4416

- [x] Shorter sentences
- [x] Explain `NAT`
- [x] Explain `SSL`

### CODE REVIEW

File: umbraco-cms/fundamentals/data/scheduled-publishing.md [hunk 1]
```diff
@@ -44,16 +44,16 @@ If you are not load balancing, the way that Umbraco determines the base URL to s
 * Else umbracoSettings:distributedCall/servers if we have the server in there _(deprecated, see load balance docs)_
 * Else it's based on the first request that the website receives and uses the base URL of this request _(default)_
 
-If the `umbracoApplicationUrl` is used, the value also specifies the scheme (either http or https). However if any of the other options are used and the appSetting `umbracoUseSSL` is set to `true`, then the request for scheduled publishing will always be sent to the HTTPS endpoint.
+If the `umbracoApplicationUrl` is used, the value also specifies the scheme (either http or https). The request for scheduled publishing will always sent over HTTPS if the appSettings `umbracoUseSSL` is set to `true`.
```
Comment (eshanrnh) [inline umbraco-cms/fundamentals/data/scheduled-publishing.md:47]:
```suggestion
If the `umbracoApplicationUrl` is used, the value also specifies the scheme (either HTTP or HTTPS). The request for scheduled publishing will always be sent over HTTPS if the appSettings `umbracoUseSSL` is set to `true`.
```

**Review by eshanrnh — APPROVED** [review]
Looks good to me, @erikjanwestendorp 🙌 Just a small suggestion though.

**Comment by erikjanwestendorp** [comment]
@eshanrnh Thanks for your review, just updated the PR with your suggestion 😄

**Comment by eshanrnh** [comment]
Looks great. Merging it in💪

**Pull request merged**